R/knitr-methods.R

```diff
@@ -20,6 +20,9 @@ registerMethods <- function(methods) {
 }
 
 .onLoad <- function(...) {
+  # warn if the version of shiny is lower than what was specified in DESCRIPTION
+  checkShinyVersion(error = FALSE)
```

Comment:
I think this will raise a NOTE with R CMD check, because it's in `.onLoad` instead of `.onAttach`.